packages/svelte/src/internal/client/dom/elements/transitions.js

```diff
@@ -309,27 +278,26 @@ export function transition(flags, element, get_fn, get_params) {
  * Animates an element, according to the provided configuration
  * @param {Element} element
  * @param {AnimationConfig | ((opts: { direction: 'in' | 'out' }) => AnimationConfig)} options
- * @param {Animation | undefined} counterpart The corresponding intro/outro to this outro/intro
+ * @param {Animation | undefined} counterpart
```

Comment:
Why was the comment removed - was it wrong? It took me quite some time to understand what this was back when I first looked at the code

Reply:
at one point I removed this argument altogether in favour of passing in `t1`, but it didn't pan out — reinstated it but forgot to add the comment back

```suggestion
 * @param {Animation | undefined} counterpart The corresponding intro/outro to this outro/intro
```